src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -171,6 +171,14 @@ JSON file, with the same label.
 Similarly the OPTIONAL `rec-<label>` key/value can be used to distinguish
 different reconstruction algorithms (for example ones using motion correction).
 
+#### The `ch` entity
+
+The OPTIONAL `ch-<index>` key/value can be used to distinguish channel-specific
+data from sequences not employing coil combination.
```

Comment:
IMO it makes sense since for MEEG we distinguish the sensors (=coil) from the channel (=from amplifier) -- this also means that across all modalities data are recorded from channels, and measured using electrodes, squids, coils, etc ..

Reply:
Useful analogy, thanks!
A little detail -- MEEG ATM does not use any `_sensor-` or `_ch-` KEYs to place individual sensors or channels into separate files. All data goes into a single data file with `_channels.tsv` to describe the content. (Please correct me if I am wrong)